packages/cli/src/commands/account/claims.test.ts

```diff
@@ -28,8 +28,8 @@ testWithGanache('account metadata cmds', (web3: Web3) => {
       writeFileSync(emptyFilePath, IdentityMetadataWrapper.fromEmpty(account))
     }
 
-    const readFile = () => {
-      return IdentityMetadataWrapper.fromFile(kit, emptyFilePath)
```

Comment:
technically fromFile was async anyawy so `readFile` would have needed to be `await`ed no mater what